triangle.json

```diff
@@ -0,0 +1,98 @@
+{
+    "#": [
+        "Pursuant to discussion in #202,",
+        "we have decided NOT to test triangles where a + b = c.",
+        "e.g: (2, 4, 2, Isosceles), (1, 3, 4, Scalene).",
```

Comment:
Reductio ad absurdum: If we're not including degenerate triangles, these tests should not be present either.